src/styles/components/organisms/Header.scss

```
@@ -0,0 +1,16 @@
+%header {
+  display: flex;
+  height: 90px;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 70px;
```

Try to use one of the spacing variables that we have for create a unit in our project.
```suggestion
  padding: 0 $space-section-big;
```

---

Ready